test/e2e/dynamic_provisioning_test.go

```diff
@@ -134,6 +134,38 @@ var _ = ginkgo.Describe("Dynamic Provisioning", func() {
 		test.Run(cs, ns)
 	})
 
+	ginkgo.It("should create a pod with volume mount subpath [file.csi.azure.com]", func() {
+		pods := []testsuites.PodDetails{
```

Comment:
add skipInlineTree test here

Reply:
Sure. Should I add the [Windows] tag in the test as well? I suppose we should test this with a windows cluster too?